inference-engine/thirdparty/movidius/mvnc/src/mvnc_api.c

```diff
@@ -1941,22 +1936,25 @@ ncStatus_t checkGraphMonitorResponse(streamId_t graphMonStream) {
     return NC_OK;
 }
 
-static void lockAllInferences() {
+static ncStatus_t lockAllInferences() {
+    GLOBAL_LOCK();
     struct _devicePrivate_t *d = devices;
     while (d) {
         CHECK_MUTEX_SUCCESS(pthread_mutex_lock(&d->graph_stream_m));
         d = d->next;
     }
-    return;
+    GLOBAL_UNLOCK();
+    return NC_OK;
 }
 
-static void unlockAllInferences() {
+static ncStatus_t unlockAllInferences() {
     struct _devicePrivate_t *d = devices;
     while (d) {
         CHECK_MUTEX_SUCCESS(pthread_mutex_unlock(&d->graph_stream_m));
         d = d->next;
     }
-    return;
+    GLOBAL_UNLOCK();
```

Comment:
No connected call of `GLOBAL_LOCK`

As we have discussed: there should be one of the following:
* There should not be `GLOBAL_UNLOCK()` in `lockAllInferences`
* GLOBAL_LOCK()/GLOBAL_UNLOCK() should be in pairs both in `lockAllInferences()` and `unlockAllInferences()` but `lockAllInferences()` should return some information about devices for which inferences have been locked